src/main/java/com/tstepnik/planner/domain/task/Task.java

```diff
@@ -24,7 +24,7 @@
     @Enumerated(EnumType.STRING)
     private Importance importance;
 
-    private Boolean isDone = Boolean.FALSE;
+    private Boolean done = Boolean.FALSE;
```

Comment:
this change doesnt belong in this PR

Reply:
lol sometimes you're comment code which is not connected with PR and now you don't want accept because of that.

Reply:
BTW you wanted it changed to "done" in one PR.

Reply:
Yeah, in one PR, not this one.